src/lib/connector.ts

```diff
@@ -75,6 +77,10 @@ export function connect(options: ConnectOptions): Connection {
     typeof options.rpcRelay === "boolean" ? options.rpcRelay : info.rpcRelay;
   // If a token was provided, we cache it
   const token = options.token;
+  const siweUri =
+    options.siweUri ??
+    globalThis.document?.location.origin ??
+    "https://tableland.xyz";
```

Comment:
Wondering if this should be testnet.tableland.xyz? cc @joewagner here as well. Now that I'm seeing this again, perhaps we actually _do_ require it be specified explicitly?

Reply:
testnet.tableland.network sounds right to me